### PR TITLE
update keygrip to version 1.0.1

### PR DESCRIPTION
**Pull request by nallown**

### CODE REVIEW

File: lib/cookies.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ Cookies.prototype.set = function(name, value, opts) {
 
   if (opts && signed) {
     if (!this.keys) throw new Error('.keys required for signed cookies');
-    cookie.value = this.keys.sign(cookie.toString())
+    cookie.value = this.keys.sign(cookie.toString()).toString('base64')
```
Comment (dougwilson) [inline lib/cookies.js]: This changes seems unrelated to updating the keygrip version. It seems very backwards-compatible as well?

**Comment by nallown** [comment]
keygrip returns a buffer now and not a string anymore which is why I convert the returned value to string now.

File: lib/cookies.js (hunk 1 above)
Comment (nallown) [inline lib/cookies.js]: keygrip returns a buffer now and not a string anymore which is why I convert the returned value to string now.

Comment (dougwilson) [inline lib/cookies.js]: What are you talking about? This module already installs 1.0.1 by default (since the package.json is ~1.0.0) and it returns a string from the sign method: https://github.com/crypto-utils/keygrip/blob/1.0.1/index.js#L12-L21

Comment (dougwilson) [inline lib/cookies.js]: I pulled down this module, which installed keygrip 1.0.1 by default, added `console.log(typeof this.keys.sign(cookie.toString()))` and it printed `string`. I also added `console.log(Buffer.isBuffer(this.keys.sign(cookie.toString())))` and it printed `false`. The `.toString('base64')` does nothing, since the `.toString()` on a string just returns the same string, and it doesn't accept any arguments. Your change here can be removed, as it's just doing `'apple'.toString('base64')`, which returns `'apple'`.

Comment (nallown) [inline lib/cookies.js]: My fault I was looking at the master branch not 1.0.1 branch. In the master branch it returns a buffer instead of a string so I thought that for v1.0.1 to work you needed to make that change.

You can close the pull request.

Comment (nallown) [inline lib/cookies.js]: I was using this test as a reference https://github.com/crypto-utils/keygrip/blob/master/test/test.js#L40
